metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1620,6 +1627,64 @@ class MetalsLanguageServer(
     results
   }
 
+  /** Requesting semantic tokens for a whole file in order to highlight */
+  @JsonRequest("textDocument/semanticTokens/full")
+  def semanticTokensFull(params: SemanticTokensParams)
+  : CompletableFuture[SemanticTokens] ={
+    scribe.info("Debug: MetalsLanguageServer.semanticHighlighting: Start")
+    CancelTokens.future { token => compilers.semanticTokens(params, token) }
+  }
+
+  val SeverSemanticTokenTypes= List(
```

Comment:
We could probably move these to an object, so it could be reused in a number of places.

Reply:
So, I will add a new object under scala.meta.internal.metals, and move the following values to it. 
Is it OK ?
  - val SeverSemanticTokenTypes
  - val SeverSemanticTokenModifiers
  - val defaultSemanticTokensServerCapability

Reply:
That should be fine, if we end up with more classes needed then we can create a separate package

Reply:
I will reflect that to new repo which is Metals folk.  https://github.com/ShintaroSasaki/metals